README.rst

```diff
@@ -1,67 +1,52 @@
 TOMATO - The Offer MAnager Tool
 -----------------------------------
 
-|maven-build| |maven-test| |release| |java| |groovy| |code-ql|
+|maven-build| |maven-test| |codeql| |release|
```

Comment:
The naming scheme has to be consistent with the linkage outlined at the end of the Readme for the badge to work
```suggestion
|maven-build| |maven-test| |code-ql| |release|
```